### DISCUSSION TITLE
Если я ничего не понял

### DISCUSSION THREAD

**Discussion by BlackPatrick** [audetv/cohesion#10] [Q&A]
Товарищи, здравствуйте!
Возьму на себя ответственность и создам обсуждение в котором можно задавать вопросы для тех, кто присоединяется и пока не оСВОил данный ресурс (я имею ввиду гитхаб) и не разобрался в разделах и темах уже существующих на terratensor.

Если я правильно все сделал то это обсуждение появится на верхнем уровне terratensor. Если я ошибся, то можно будет перенести ее.

**Comment by BlackPatrick** [comment]
Я задам первый вопрос. Когда только присоединился, то мне попалась тема с обсуждением по выбору названия terratensor, но теперь не могу ее найти. Помогите, пожалуйста!

**Reply by audetv** [reply]
Я думаю это тут:
https://github.com/terratensor/cohesion/issues/2

В разделе репозитория cohesion в issues

**Reply by iprst** [reply]
terra — geo 
tensor — matrix

терра == гео
тензор это матрица с количеством измерений большим, чем два

**Reply by BlackPatrick** [reply]
Да, оно. Спасибо.

**Reply by BlackPatrick** [reply]
> terra — geo tensor — matrix
> 
> терра == гео тензор это матрица с количеством измерений большим, чем два

Это я понял. Хотел еще раз почитать логику по которой Вы пришли к этому названию. Алексей уже ответил мне. Спасибо.

**Comment by BlackPatrick** [comment]
Следующий мой вопрос. Какие-то правила по обсуждаемым здесь темам есть? Какова изначальная цель для создания этого ресурса?
Здесь можно поднимать только тема касаемые чего-то конкретного, например, связанного с проектом svodd.ru или любые связанные с КОБ и ДОТУ?

**Reply by audetv** [reply]
> Какие-то правила по обсуждаемым здесь темам есть?

Мы не ограничивали себя набором обсуждаемых тем, темы зависят от личной меры участников и выдвигаются их произволом. В геоматрице мы, например, вообще обсуждаем какие-то еще не названные процессы мироздания)

> Какова изначальная цель для создания этого ресурса?

Начиналось так: [Концепция связности #4](https://github.com/terratensor/cohesion/issues/4#issue-1751005200)
Работа над геоматрицей, все проекты (репозитории), которые тут созданы связаны с геоматрицей. 
https://github.com/terratensor/geomatrix

> Здесь можно поднимать только тема касаемые чего-то конкретного, например, связанного с проектом svodd.ru или любые связанные с КОБ и ДОТУ?

Можно обсуждать любые темы, и связанные с КОБ и ДОТУ, но возможно, что какие-то темы имеет смысл поднимать в соборной теме, это на усмотрение автора.

**Reply by iprst** [reply]
> Какие-то правила по обсуждаемым здесь темам есть?

Специальных не вводили, общие есть — не нарушать законы и тд тп. 

> Здесь можно поднимать только тема касаемые чего-то конкретного, например, связанного с проектом svodd.ru или любые связанные с КОБ и ДОТУ?

Сейчас нет перечня того, что связано, а что не связано со svodd.ru кроме того это даже не единственный наш домен. Пока связь сущностей не описана в достаточной мере, но можно пробежаться по issue как закрытым, так и открытым, и в целом будут понятны направления проектов, чтобы иметь представление о том что связано, а что нет. Кобовские темы прямо не касающиеся технологических решений, в принципе наверное полезнее обсуждать на ФКТ, здесь в основном у нас затрагиваются практические реализации методов, и соответственно методология с теорией, связанная с ними.

**Reply by BlackPatrick** [reply]
Отлично. Я в целом так и предполагал. Я спросил по той причине, что я с профессиональной точки зрения (я метролог по образованию и по профессии) начал заниматься реализацией проекта по метрологическому обеспечению цифровых моделей. В качестве основы я конечно же применяю ДОТУ. Почитав немного здешние проекты, решил, что можно было бы здесь создать ветку для обсуждения этого проекта. Правда участников тут не так много, но может быть это только начало, но зато я увидел. что вы программисты, а это та область где я не обладаю необходимыми компетенциями и мне будет важно взгляд на этот вопрос с точки зрения ИТ-профессионалов.

Заодно будет у меня некое информационное пространство, где можно будет консолидировать накопленные знания.

**Reply by BlackPatrick** [reply]
> Пока связь сущностей не описана в достаточной мере, но можно пробежаться по issue как закрытым, так и открытым, и в целом будут понятны направления проектов, чтобы иметь представление о том что связано, а что нет.

Да, сейчас как раз по мере возможности занимаюсь изучением тут материалов

**Reply by iprst** [reply]
> участников тут не так много, но может быть это только начало

Пока начало, но причина небольшого количества участников не в этом. Мы изготавливаем остриё методологии, притом исходя из гипотез, которые не являются сколько-нибудь принятыми где бы то ни было. Возьмите хоть поисковик по текстам книг, хоть геоматрицу — системы работают, и кажется что исходят их разных предпосылок, но на самом деле нет, предпосылка одна. Поиска по текстам книг с выдачей результатов как связных текстов не существовало, как не существовало поиска по географическим объектам с выдачей результата связей в форме текста. Можно найти ответ на вопрос «о чём эта книга и процитируй страницу 234», можно найти ответ на «какие самые густонаселённые города планеты», но нельзя найти ответ на «перечисли параграфы из тысяч книг, где упомянут конь бозоны двенадцать синяя» или на «Тамерлан против острова Пасхи». Между тем именно эти вопросы намного интереснее, чем цитата 234 страницы и списка милионников. Пока так объясню. 

> Почитав немного здешние проекты, решил, что можно было бы здесь создать ветку для обсуждения этого проекта.

Если вы полагаете, что ваши идеи как-то сочетаются или могут в теории сочетаться с тем, что представлено в терратензоре (лично я не представляю идеи, которая _не сочетается_, но это моё личное мнение), а ещё лучше — расширяют или пополняют функционал существующих и разрабатываемых решений, то можно открыть issue в любом подходящем вашей теме репозитории, там и обсудим. Просто напишите всё, что считаете нужным, в начальном посте, добавьте если требуется в комментариях под ним, и будем разбираться. 

> некое информационное пространство, где можно будет консолидировать накопленные знания.

Это самое полезное из.

**Reply by BlackPatrick** [reply]
> Мы изготавливаем остриё методологии, притом исходя из гипотез, которые не являются сколько-нибудь принятыми где бы то ни было.

В метрологии, в части цифровых моделей, тоже самое примерно. Есть ряд решений, но они все половинчатые, решают только какую-то часть задачи. Или имеющиеся решения разрознены и в некоторых случаях эти разрозненные решения могут или дублировать друг друга с некоторыми особенностями или противоречить друг другу. Кроме того, метрологическое научное сообщество с отторжением воспринимает предложения о том, что некое ПО может выполнять функцию средства измерений. Средство измерений в их парадигме обязательно должно быть физическим, как и передача единицы величины, должны быть обязательно от физического эталона к физическому средству измерений. Вот и возникла необходимость в разработке новой концепции метрологического обеспечения цифровых моделей. 

> Возьмите хоть поисковик по текстам книг, хоть геоматрицу — системы работают, и кажется что исходят их разных предпосылок, но на самом деле нет, предпосылка одна.

Да, предпосылка одна. Концепция связности.

> можно открыть issue в любом подходящем вашей теме репозитории, там и обсудим

Отлично, ознакомлюсь подробнее с имеющимися материалами, подготовлю свои и создам тему для обсуждения.

**Reply by iprst** [reply]
> В метрологии, в части цифровых моделей, тоже самое примерно. Есть ряд решений, но они все половинчатые, решают только какую-то часть задачи. Или имеющиеся решения разрознены и в некоторых случаях эти разрозненные решения могут или дублировать друг друга с некоторыми особенностями или противоречить друг другу.

Такие высокоуровневые задачи очень хорошо решаются переносом их решения на какую-то узкую специальную область, которая выступает рабочей моделью. Многие принципы предельного, глобального объединения сущностей возникают на более элементарных уровнях анализа. Я например почти всё тестирую на программах изготовления и анализа музыки, то есть преследую тезис, что любую задачу можно свести к аналогу теории обработки сигналов. Тезис считаю верным, но не уникальным — в пределе решение любой задачу можно свести к решению уже известной, если найти подходящий оракул. Геоматрица это вычислительный оракул. Поисковик по текстам книг это вычислительный оракул. ДОТУ это вычислительный оракул. Программы для производства и анализа звука это вычислительный оракул. И миллионы других подобных. Так что вам для вашего решения понадобится найти уже существующую систему решений, и оракул — метод перехода от одной системы к другой без потери логической связности. Как говорим на  ФКТ — натягивайте на ДОТУ. Я бы уточнил — натягивается на триединство. Это истинный ключ всей логики ДОТУ, незаслуженно игорируемый. 

> Кроме того, метрологическое научное сообщество с отторжением воспринимает предложения о том, что некое ПО может выполнять функцию средства измерений. Средство измерений в их парадигме обязательно должно быть физическим, как и передача единицы величины, должны быть обязательно от физического эталона к физическому средству измерений. Вот и возникла необходимость в разработке новой концепции метрологического обеспечения цифровых моделей.

Сопротивление понятно и тому помимо очевидных причин есть неочевидные. Неочевидные на мой взгляд даже сильнее — появились ИИ системы, которые легко попирают любые прошлые метрологические принципы. Но тут исключительно требуется начинать с триединства. Без него просто будет шалтай-болтай. Делайте issue, там и начнём разбирать по возможности ваши наблюдения.

**Reply by BlackPatrick** [reply]
Именно так и делаю. У меня уже есть конкретный техпроцесс, на который имеется действующая цифровая модель. Сейчас работаем над тем, чтобы на нее все и наворачивать. И что касается ДОТУ и триединства, то именно на них и натягиваю. Обязательно создам issue, а может даже и отдельный repository на эту тему и перейду к конкретным вещам.

По сопротивлению, то здесь у меня такие же наблюдения и выводы. Самое интересное, что они всячески пытаются игнорировать такие системы. Мол, ИИ это что-то айтишное и не надо сюда приплетать науку. Там где нет фундаментальных принципов, говорят они, там нет метрологии. Все это всего лишь вычисления в виде ПО и все решения для подтверждения корректности вычислений ПО уже есть. Изобретать велосипед на их взгляд не нужно.

**Reply by iprst** [reply]
> Там где нет фундаментальных принципов, говорят они, там нет метрологии.

Логично. Ведь масло масляное и это невозможно опротестовать. Ведь есть универсальная линейка, позволяющая проверить сущность на содержание в ней фундаментальных принципов и указать, что в сущности фундаментальных принципов не содержится, а посему она не требует рассмотрения. Приложил, проверил, не содержит, следующий. 

> ИИ это что-то айтишное и не надо сюда приплетать науку

Восхитительно. Это специальный подвид предельно обобщающей категории «это другое». А что, всё метрологически состоятельно. Приложил, проверил, это что-то другое, следующий. А когда утверждается, что к чему-то не надо приплетать науку, можно сделать медицинский вывод — это что-то уже окучивается сектой. 

> Все это всего лишь вычисления в виде ПО и все решения для подтверждения корректности вычислений ПО уже есть.

Тут вам в руки нужно брать теорию сложности вычислений и теорию квантовой сложности, прямо в них брать формулы расчёта сложности, и показывать, чем обернётся подтверждение корректности вычислений в сложных нейросетях на обычном и на квантовом компьютере. Возможно для этого даже хватит времени существования Вселенной. Но тут в ход вступает вторая предельно обобщающая категория — «но это не точно».

**Reply by BlackPatrick** [reply]
> Следующий мой вопрос. Какие-то правила по обсуждаемым здесь темам есть? Какова изначальная цель для создания этого ресурса? Здесь можно поднимать только тема касаемые чего-то конкретного, например, связанного с проектом svodd.ru или любые связанные с КОБ и ДОТУ?

Продублирую здесь мое предложение по ПФУ цифровых моделей. Если данная тема в этой ветке актуальна и найдутся желающие для дальнейшей работы в этом направлении, то можно будет здесь создать отдельный репозиторий или issue. Если она тут неуместна, то продумаю другое место для активности.

Учитывая, что ИИ - это инструмент управления, то этот инструмент нам, в русском мире, ни в коем случае нельзя оставлять без внимания. Это же касается в том числе и различных цифровых моделей и цифровых двойников, которые массово внедряются в нашу жизнь. Исходя из этого, я рассмотрел данный процесс по полной функции управления и предлагаю всем желающим присоединиться к рассмотрению данной ПФУ. чтобы снять различные субъективизмы, выраженные мной через мою личную меру понимания. В дальнейшем убежден, что среди нас найдутся участники, которые имеют прямое отношение в своей жизни (профессионально или личной) как к цифровым моделям, так и смежным процессам и объединив усилия мы сможем с нашего уровня решить масштабную задачу, описанную по ПФУ. Задача высокоуровневая и для ее решения потребуется государственное участие (имеется ввиду, что потребуются изменения в государственно реализуемых и контролируемых процессах, например, законодательных) и поэтому при реализации мы столкнемся с различными ограничениями. Но практика показывает, что подобные задачи решаются и успешно. Фактором успешности является отработка данной задачи на каком-либо реальном примере и последующая демонстрация результатов. У меня уже есть реальный проект, в рамках которого я уже реализую предложенный подход, но один в поле не воин и буду рад, объединить с кем-либо усилия.

Ниже привожу ПФУ (в ПФУ использовал цитаты из ДОТУ без указания на это). Тема обширная, поэтому нужно будет рассмотреть отдельный ресурс для ее обсуждения и реализации, чтобы не смешивать с обсуждением здесь. Как вариант можно будет на гитхабе создать отдельный репозиторий на терратензоре. Или создать свою отдельную тему на гитхабе. Или может быт это будет совсем другой ресурс. Тут предложения могут быть различные. В общем я начну тему, и если будет актуально, то организуемся. Поехали.

ПФУ цифровых моделей

### 1. Выявление фактора среды, который «давит на психику», чем и вызывает субъективную потребность в управлении

Управление на основе виртуальных структур. Это — тоже один из процессов, возможных в суперсистемах. Мироздание предстаёт в качестве объемлющей суперсистемы по отношению ко множеству взаимно вложенных суперсистем со структурой, изменяющейся в каждый момент времени, а кроме того, — и определяемой разными субъектами по разным наборам признаков. Такого рода объективная множественность и субъективная неопределённость в вопросе об идентификации каждой из суперсистем и её структуры даёт основание к тому, чтобы назвать все такого рода структуры «виртуальными».

К «виртуальным» можно отнести и такие структуры, как различные цифровые модели объектов, процессов и явлений. Частным случаем такой модели иерархически наивысшего порядка является Искусственный Интеллект.

Взаимная вложенность суперсистем предполагает существование элементов, одновременно принадлежащих к нескольким суперсистемам. В данном случае термин «одновременная принадлежность» означает, что в памяти элемента, одновременно принадлежащего разным суперсистемам из числа нескольких взаимно вложенных, присутствуют фрагменты информационно-алгоритмического обеспечения каждой из суперсистем этого множества. Хотя в один и тот же момент времени этот элемент вовсе не обязательно занят в отработке алгоритмики каждой из суперсистем, к которой он принадлежит. Т.е. виртуальность структур предполагает существование элементов, в разные моменты времени проявляющих активность в алгоритмике разных суперсистем под воздействием внутренних процессов адресного обмена информацией в каждой из них (такого рода адресного распространения информации нет в бесструктурном управлении, которое возникает как результат циркулярного безадресного распространения информации).

В отношении элементов ИИ суперсистемами в его составе выступают различные информационно-алгоритмические модули.

Приведём пример управления на основе виртуальных структур. Предположим, что:
- «суперсистема № 1» представляет собой подмножество элементов «суперсистемы № 2», т.е. всякий элемент «суперсистемы № 1» является одновременно и элементом «суперсистемы № 2», но не всякий элемент «суперсистемы № 2» является элементом «суперсистемы № 1»;
- «суперсистема № 2» не видна с уровня «суперсистемы № 1»;
- в «суперсистему № 1» проникает структура, несущая некий процесс управления, организованная на уровне «суперсистемы № 2».

При оговоренных условиях, функционирование этой структуры будет восприниматься на уровне «суперсистемы № 1» как ничем не обусловленные «случайные совпадения» взаимодополняющего характера в поведении элементов «суперсистемы № 1», но не как проявление деятельности структуры, проникающей в «суперсистему № 1» из «суперсистемы № 2».
Если природа этих не случайных совпадений на уровне «суперсистемы № 1» не может быть выявлена, то на уровне «суперсистемы № 2» вся такого рода совокупность «случайных» совпадений — управление на основе виртуальных структур.

Это — один из примеров управления на основе виртуальных структур. В более общем случае любое проникновение структурного управления в суперсистему извне — в ней предстаёт как управление на основе виртуальных структур вне зависимости от того, проникает в суперсистему структурное управление из иерархически равнозначной ей суперсистемы, либо это иерархически высшее управление, вплоть до иерархически наивысшего всеобъемлющего управления Вседержителя. При таком понимании любое бесструктурное управление — частный случай управления на основе виртуальных структур, осуществляемого Вседержителем.

Структурное управление в суперсистемах может возникать как реализация соответствующего этапа полной функции управления — целенаправленное построение структуры, несущей концепцию управления. Но структурное управление в суперсистемах может возникать и из бесструктурного или из управления на основе виртуальных структур, если цели, на которые ориентировалось бесструктурное управление или управление на основе виртуальных структур, достаточно устойчивы, вследствие чего обретают устойчивость и структуры, сложившиеся в бесструктурном управлении или в виртуальном управлении для работы с этими целями.

Иными словами, структурное управление может выкристаллизовываться из бесструктурного или из управления на основе виртуальных структур.

Наивысшее качество управления в суперсистемах достигается в сочетании структурного и бесструктурного управления в русле адекватного иерархически высшего управления, протекающего в них на основе виртуальных структур в ладу со Вседержительностью.

Таким образом на основе ИИ, как виртуальной структуры, может осуществляться бесструктурное управление толпой с иерархически высшего уровня управления, протекающего ни как ни в русле Божьего промысла, а соответствующее Библейской концепции.

Учитывая то, что ИИ как и любая другая виртуальная структура состоит из элементов низшего порядка (информационно-алгоритмических модулей), то применение ИИ для целей управления с помощью него в ладу со Вседержительностью сводится к управлению этими самыми элементами.

Информационно-алгоритмические модули являются цифровыми моделями, описывающими объекты, процессы и явлений и соответственно на сегодняшний день фактором среды, который «давит на психику» и вызывает субъективную потребность в управлении, является потребность управлять цифровыми моделями с целью направления применения таких моделей в русло Божьего промысла.

### 2. Формирование навыка (стереотипа) распознавания фактора среды на будущее и распространение его в культуре общества.

Рассмотрим применение цифровых моделей на примере их применения в научно-производственной, социальной, творческой или иной деятельности человека. Использование цифровых моделей в деятельности человека позволяет исследовать окружающий мир, восстанавливать события прошлого, разрабатывать технологии, симулировать процессы, оптимизировать работу оборудования, предотвращать поломки и аварии, тестировать продукты, проигрывать разные варианты развития событий. Некоторые сферы, в которых применяется цифровое моделирование, кажутся далекими от промышленного производства, но и в них эта технология помогает усовершенствовать процессы, что в итоге способно поменять всю экономику отрасли. Например, социологические исследования с применение цифровых моделей позволяют получить более глубокий анализ состояния общества.

В школах, вузах и на производствах цифровое моделирование упрощает обучение. При разработке месторождений, строительстве, реализации экологических проектов цифровые модели помогают просчитать последствия, прогнозировать стихийные бедствия, учитывать предыдущий опыт при конструировании сооружений.

Особую роль в современном производстве играют цифровые двойники – виртуальные копии физических устройств, моделирующие их поведение в реальных условиях. С их помощью можно разрабатывать новые технологические схемы производства, выбирать оптимальный режим и состав сырья, оценивать экономический эффект, заранее «проигрывая» работу объекта. Они помогают принимать оптимальные решения в условиях сложных многопараметрических процессов, а иногда полностью исключить человеческий фактор.

Если в этом многообразии нашей деятельности мы все больше доверяем управлению с помощью цифровых моделей, то уже сегодня это сформировало стереотип распознавания необходимости предъявление определенных требований к этим моделям.

### 3. Целеполагание в отношении выявленного фактора

В зависимости от того, какие будут сформированы требования к цифровым моделям, применяемым человеком в своей деятельности, будет зависеть то, будут ли их применение лежать в русле Божьего промысла или же будет вписана в общий вектор целей в рамках Библейской концепции.

Вектором целей для ПФУ в отношении любых цифровых моделей в этом случае будет определение требований к этим моделям и реализация их на практике.

### 4. Формирование генеральной концепции управления и частных концепций управления в отношении каждой из целей в составе вектора целей

Формирование концепции управления и частных целевых функций управления, составляющих в совокупности концепцию, на основе решения задачи об устойчивости в смысле предсказуемости поведения.

Поскольку информационное обеспечение самоуправления элементов в своей фундаментальной части предполагает некоторое взаимодействие и информационный обмен между элементами суперсистемы, а информационное обеспечение их самоуправления в своей адаптационной части формируется разнообразно, но статистически упорядочено, то в суперсистеме всегда существует вероятностно предопределённая возможность того, что:
1. Некий элемент (интеллект или автомат) распознаёт фактор среды, с коим сталкивается.
2. Распространит информацию о нём.
3. Информация будет принята элементами, обладающими в своей памяти необходимой концепцией управления или способными построить её.
4. Концепция управления будет передана инициатору порождения структуры.
5. Найдутся элементы, свободные или занятые в структурах с менее важными приоритетами, обладающие специализацией, необходимой для вновь создаваемой структуры и так далее, вплоть до того, что в приемлемые сроки процесс управления свершится по полной функции с приемлемым уровнем качества.

Исходя из того, что внедрение цифровой модели направлено на решение задачи управления, то необходимо обеспечить функционирование цифровой модели по полной функции управления, при этом алгоритмику реализации каких-либо этапов ПФУ цифровой модели субъект-управленец берёт на себя, а каких-то возлагает на цифровую модель в режиме ее самоуправления объектом.

Таким образом, для реализации заданного вектора целей необходимо разработать систему нормативно-правовой документации. устанавливающей единые требования к цифровым моделям. Требования, которые будут обеспечивать реализацию в цифровых моделях алгоритмов ПФУ. В дальнейшем необходимо будет внедрение данных требований в систему стандартизации.

При разработке и внедрении цифровых моделей необходимо предусматривать функционирование модели исходя из того, что объект управления в цифровой модели должен ей рассматриваться в трех динамических состояниях. Первое состояние – это текущее состояние, в котором находится объект. Второе состояние – это целевое состояние, которого мы хотим достичь при управлении этим объектом. И третье состояние – это состояние, обозначающее разницу между первым и вторым состоянием с учетом их неопределенностей – контрольное состояние. В идеале задача управления объектом сводится к тому, чтобы контрольное состояние свести к нулю, но так как для любого из состояний по каждому из параметров имеются неопределенности, то задача управления сводится к тому, чтобы текущее состояние с учетом этих неопределенностей, не выходило за границы допустимой неопределенности для целевого состояния.

Решение прикладной задачи управления без выбора иерархически упорядоченного набора контрольных параметров, по изменению которых можно судить о качестве управления, невозможно. Само же управление осуществляется с помощью параметров, на которые непосредственно возможно воздействие. Такие параметры называются управляемые. При их изменении меняются и контрольные параметры. Помимо этих двух перечней, в ходе управления необходимо, также, учитывать перечень свободных параметров, которые не изменяются при изменении параметров управления, либо изменяются, но не входят в перечень контрольных параметров.

Каждый из параметров, входящих в эти перечни, должен быть либо измеримым, либо численно определяемым. Под численно определяемыми параметрами понимаются параметры, количественные характеристики которых получены не путем измерений в ходе осуществления управления, а на основании ранее проведенных исследований или статистических и справочных данных, и принимаемых в момент управления постоянными или условно постоянными. Измеряемые параметры – это параметры, количественные характеристики которых получены путем измерений непосредственно в ходе осуществления управления. Если каждый из параметров не удается описать количественно, то такая модель утрачивает устойчивость. Но только определение перечней всех параметров не является полной и достаточной мерой, позволяющей говорить об устойчивости модели управления. Устойчивость модели определяется полным перечнем параметров (контрольные, управляемые, свободные), каждый из которых идентифицирован, измерен или численно определен и для каждого из параметров установлены границы допустимого отклонения для всего возможного числа его состояний при различных сочетаниях с остальными параметрами.
Следующим шагом для достижения устойчивости цифровой модели необходимо решить задачу установления для каждого из параметров границ допустимого отклонения для всего возможного числа ее состояний при различных сочетаниях с остальными параметрами.

Рассматривая реализацию такого похода к цифровым моделям на примере любого технологического процесса, то технологический процесс разбивается на отдельные этапы, для каждого из которого может быть реализована своя нижестоящая цифровая модель и для каждой такой модели применяются единые требования к ней. Когда для каждого из этапов технологического процесса будет реализована цифровая модель, то для всего технологического процесса будет возможным определить полный перечень контролируемых, управляемых и свободных параметров, и устойчивость верхнеуровневой цифровой модели управления всем технологическим процессом будет определяться устойчивостью цифровых моделей для каждого этапа в различных их состояниях. Таким образом задача по достижению устойчивости становиться обязательной для любых цифровых моделей. Это высокоуровневая задача и ее решения необходимо перенести в конкретную специализированную область и отработать ее на каком-либо реальном примере. Такой специализированной областью является область метрологии. Именно в рамках метрологического обеспечения должна решаться такая задача. Требования к цифровым моделям должны стать обязательными в области обеспечение единства измерений.

### 5. Внедрение генеральной концепции управления в жизнь — организация новых или реорганизация существующих управляющих структур, несущих целевые функции управления

Организация и реорганизация целесообразных управляющих структур, несущих целевые функции управления
Внедрение разработанной концепции управления цифровыми моделями в жизнь необходимо осуществлять через внедрение требований в систему стандартизации, путем разработки серии стандартов, норм и правил, обеспечивающих их реализацию, а также разработки концепции метрологического обеспечения цифровых моделей.

### 6. Контроль (наблюдение) за деятельностью структур в процессе управления

В процессе реализации могут быть созданы отдельные рабочие группы экспертов на базе различных институтов, осуществляющих координацию при реализации поставленного вектора целей. Контроль за внедрение разработанных требований к цифровым моделям необходимо осуществлять через систему сертификации.

### 7. Совершенствование действующей концепции 

В случае необходимости будет осуществляться в ходе реализации вектора целей созданными рабочими группами и их регулярного взаимодействия.

### 8. Ликвидация существующих структур и высвобождение используемых ресурсов в случае ненадобности либо поддержание их в работоспособном состоянии до следующего использования

Ликвидация созданных рабочих групп после создания системы стандартизации цифровых моделей, которая в последующем должна функционировать и совершенствоваться в режиме самоуправления.

Возможно поддержание рабочих групп в работоспособном состоянии для выполнения задач совершенствования системы стандартизации, сертификации

**Reply by borisevan923-a11y** [reply]
Интересный и очень глубокий методологический подход к ПФУ цифровых моделей. Действительно, если рассматривать ИИ и цифровые двойники не просто как изолированное прикладное ПО, а как элементы управления вложенных суперсистем, то на первый план выходит именно верификация их устойчивости и метрологическая аттестация алгоритмов. Ограничение исключительно «физическими эталонами» в традиционной метрологии сегодня сильно тормозит стандартизацию виртуальных структур. При этом на этапе практической отработки таких моделей (например, при сведении контрольных параметров к минимуму неопределенности) разработчики часто упираются в доступность распределенных вычислительных мощностей. Чтобы не замыкаться на локальном железе и протестировать концепцию на разных архитектурах, удобно задействовать готовые внешние платформы. В этом контексте можно использовать [ranvik.ru](https://ranvik.ru/chat) как единую экосистему, объединяющую ChatGPT, Claude, Grok и DeepSeek без инфраструктурных барьеров. Это позволяет оперативно сопоставлять логику генерации кода и математических вычислений на разных движках в реальном времени. Возвращаясь к вашей концепции: перенос задачи в область теории сложности вычислений и её увязка с триединством — это, пожалуй, единственный способ создать жизнеспособную нормативную базу, которая сможет подтверждать корректность динамических ИИ-систем на государственном уровне. Готов подключиться к обсуждению, если решите вынести это в отдельный специализированный репозиторий.

**Reply by iprst** [reply]
> на этапе практической отработки таких моделей (например, при сведении контрольных параметров к минимуму неопределенности) разработчики часто упираются в доступность распределенных вычислительных мощностей

Здесь баланс очень простой — никакой неопределённости в реальном процессе нет вовсе, а системы на основе нейросетей буквально порождают эту неопределённость целенаправленно для увеличения потребляемых физической системой ресурсов. Основной инструмент для такого искусственного порождения неопределённости это специальный раздел математики (нет, не теорвер), в котором уютно устроилась функция softmax (и все её аналоги): псевдонаучный квазиматематичекий релятивизм встроен в архитектуру трансформеров на уровне идеологии, как математическая неизбежность: softmax не умеет выдавать ноль или единицу, он всегда оставляет «зазор для сомнения».

> перенос задачи в область теории сложности вычислений и её увязка с триединством — это, пожалуй, единственный способ создать жизнеспособную нормативную базу, которая сможет подтверждать корректность динамических ИИ-систем на государственном уровне

Что такое «динамические ии-системы»?

**Comment by audetv** [comment]
> Возьму на себя ответственность и создам обсуждение в котором можно задавать вопросы для тех, кто присоединяется и пока не оСВОил данный ресурс (я имею ввиду гитхаб) и не разобрался в разделах и темах уже существующих на terratensor.

Все правильно, это дело, поддерживаю!

> Если я правильно все сделал то это обсуждение появится на верхнем уровне terratensor. Если я ошибся, то можно будет перенести ее.

Да все так, сообщение на верху дискуссии.

**Reply by BlackPatrick** [reply]
Отлично
